assets/test.yaml

```diff
@@ -13,17 +12,17 @@ kmer_length: 7
 dimensionality_reduction_methods: ["pca", "random_trees"]
 # all available methods
 # "pca,umap,t-sne,isomap,lle_standard,lle_hessian,lle_modified,mds,se,random_trees,kernel_pca,pca_svd,autoencoder_sigmoid,autoencoder_linear,autoencoder_selu,autoencoder_relu,nmf"
-nt_database_path: /lustre/scratch123/tol/teams/tola/users/ea10/pipeline_testing/20240704_blast_tiny_testdb/blastdb/
-nt_database_prefix: tiny_plasmodium_blastdb.fa
-nt_kraken_database_path: /nfs/treeoflife-01/teams/tola/users/dp24/ascc/kraken2/kraken2/
-ncbi_taxonomy_path: /lustre/scratch123/tol/resources/taxonomy/latest/new_taxdump
-ncbi_ranked_lineage_path: /lustre/scratch123/tol/resources/taxonomy/latest/new_taxdump/rankedlineage.dmp
-busco_lineages_folder: /lustre/scratch123/tol/resources/busco/latest/
-busco_lineages: "diptera_odb10,insecta_odb10"
-fcs_gx_database_path: /lustre/scratch124/tol/projects/asg/sub_projects/ncbi_decon/0.4.0/gxdb/
-vecscreen_database_path: /nfs/treeoflife-01/teams/tola/users/dp24/ascc/vecscreen/
-diamond_uniprot_database_path: /lustre/scratch123/tol/teams/tola/users/ea10/pipeline_testing/20240704_diamond_tiny_testdb/ascc_tinytest_diamond_db.dmnd
-diamond_nr_database_path: /lustre/scratch123/tol/teams/tola/users/ea10/pipeline_testing/20240704_diamond_tiny_testdb/ascc_tinytest_diamond_db.dmnd
+nt_database_path: /lustre/scratch122/tol/resources/nt/latest/
+nt_database_prefix: nt
+nt_kraken_database_path: /lustre/scratch122/tol/resources/nt_kraken2/latest/
+ncbi_taxonomy_path: /lustre/scratch122/tol/resources/taxonomy/latest/new_taxdump/
+ncbi_ranked_lineage_path: /lustre/scratch122/tol/resources/taxonomy/latest/new_taxdump/rankedlineage.dmp
+busco_lineages_folder: /lustre/scratch122/tol/resources/busco/latest
+busco_lineages: "metazoa_odb10"
+fcs_gx_database_path: /tmp/tol_data/fcs-gx/2023-01-24/
+vecscreen_database_path: /nfs/treeoflife-01/teams/tola/users/dp24/ascc/ascc_vecscreen
+diamond_uniprot_database_path: /lustre/scratch122/tol/resources/uniprot_reference_proteomes/latest/reference_proteomes.dmnd
+diamond_nr_database_path: /lustre/scratch122/tol/resources/nr/latest/nr.dmnd
```

Comment:
Should this yaml file be called "test_farm" or something to indicate to external users that it's for internal use?